common/beacon.go

```diff
@@ -6,6 +6,11 @@ import "os"
 // value should not be changed for backward-compatibility reasons
 const DefaultBeaconID = "default"
 
+// DefaultChainHash is the value used when chain hash has an empty value on requests
+// from clients. This value should not be changed for
+// backward-compatibility reasons
+const DefaultChainHash = "default"
```

Comment:
Are you using this ? ctrl C ctrl F on the PR doesn't show anything. If not then let's remove it.

Reply:
Yes, we are using it! I decided to have two different default variables even though their values are the same in case we want to set them with different values. Their purposes are not the same.